appinventor/buildserver/src/com/google/appinventor/buildserver/ProjectBuilder.java

```diff
@@ -315,8 +315,10 @@ public InputStream getInput() throws IOException {
       throws IOException, JSONException {
     Map<String, String> nameTypeMap = Maps.newHashMap();
 
-    JSONArray simpleCompsJson = new JSONArray(Resources.toString(ProjectBuilder.
-        class.getResource("/files/simple_components.json"), Charsets.UTF_8));
+    JSONArray simpleCompsJson = new JSONObject(
```

Comment:
Not important, just gets things to compile since I changes simple_components.json